protozfits/tests/test_R1_example_file.py

```diff
@@ -13,20 +14,16 @@
 )
 
 
-def test_can_open_file():
-    from protozfits import SimpleFile
-    SimpleFile(example_file_path)
-
-
 def test_can_iterate_over_events_and_run_header():
-    from protozfits import SimpleFile
-    f = SimpleFile(example_file_path)
 
-    camera_config = next(f.CameraConfig)
-    assert (camera_config.expected_pixels_id == np.arange(14)).all()
+    with SimpleFile(example_file_path) as f:
+
+        camera_config = next(f.CameraConfig)
```

Comment:
why do we have access the header with `next()` ?

Reply:
https://github.com/cta-sst-1m/protozfitsreader#runheader

Reply:
ok then is it worth adding a # TODO in the code ? or a Xfail test if one tries to access the header directly ?

Reply:
I do not understand what you mean by "access the header directly"

Reply:
Doing : 
```python
camera_config = f.CameraConfig
print(camera_config.expected_pixels_id) 
```